MIGRATING.md

```diff
@@ -61,6 +61,45 @@ major releases of `cosmwasm`. Note that you can also view the
   Note that this means you can mix "messages" and "submessages" in any execution
   order. You are no more restricted to doing "submessages" first.
 
+- Rename the `send` field to `funds` whenever constructing a `WasmMsg::Execute`
+  or `WasmMsg::Instantiate` value.
+
+  ```diff
+    let exec = WasmMsg::Execute {
+        contract_addr: coin.address.into(),
+        msg: to_binary(&msg)?,
+  -     send: vec![],
+  +     funds: vec![],
+    };
+  ```
+
+- `Uint128` field can no longer be constructed using a struct literal. Call
+  `Uint128::new` (or `Uint128::zero`) instead.
```

Comment:
Looks good, also providing sed commands helps a lot at the end of the document. 
Like:
- `Uint128(0) -> Uint128::zero()`
- `Uint128( -> Uint128::new(`